packages/server-node/src/launch/socket-server-launcher.ts

```diff
@@ -30,11 +30,24 @@ export class SocketServerLauncher extends GLSPServerLauncher<net.TcpSocketConnec
 
     run(opts: net.TcpSocketConnectOpts): Promise<void> {
         const netServer = net.createServer(socket => this.createClientConnection(socket));
+
         netServer.listen(opts.port, opts.host);
-        this.logger.info(`The GLSP server is ready to accept new client requests on port: ${opts.port}`);
-        // Print a message to the output stream that indicates that the start is completed.
-        // This indicates to the client that the sever process is ready (in an embedded scenario).
-        console.log(this.startupCompleteMessage);
+        netServer.on('listening', () => {
+            const addressInfo = netServer.address();
+            // eslint-disable-next-line no-null/no-null
+            if (addressInfo === null) {
+                this.logger.error('Could not resolve GLSP Server address info');
+                return;
```

Comment:
We should probably call `this.shutdown()` as we don't expect anyone to connect in this case

Reply:
I get this message when I add `this.shutdown()`: _Could not stop launcher. Launcher is not running!_
It seems for me that in run(), [this.running](https://github.com/eclipse-glsp/glsp-server-node/blob/35b3b9dff6b49707605c479827cb9debbed3079f/packages/server-node/src/launch/glsp-server-launcher.ts#L44) should be set to 'true'.

Reply:
To test that, I ran the "Debug Workflow Example Server" target, with a breakpoint on line 36 of packages/server-node/src/launch/socket-server-launcher.ts, e.g. `const addressInfo = netServer.address();`
If adressInfo is changed to a string, we can run into the else if [String] branch and test shutdown.

Reply:
This has been updated in next commit (invert  order of running).